VTUL/data-repo
# Gem updates 4.2.11

**Upgrade rails version to latest.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1672) (:star:)

# What does this Pull Request do? (:star:)
Upgrades Gemfile to use Rails 4.2.11 to address security issues as well as the missing Logger gem issue. (replaces PR's #243 and #242 )

# How should this be tested?
* Build `gem_updates_4.2.11` branch
* Ensure that build is successful
* Ensure that the following gem versions are >= these listed versions:
rack (1.6.11)
loofah (2.2.3)
rubyzip (1.2.2)
sprockets (3.7.2)
actionview (4.2.10)
activerecord (4.2.10)
* Browse through UI and check that functionality is unaffected


# Additional Notes:
* branch: `gem_updates_4.2.11`

# Interested parties
@pmather 

(:star:) Required fields
